tests/generate/test_generate.py

```diff
@@ -144,17 +206,44 @@ def test_generate_fsm(request, model_fixture, pattern):
 
     model = request.getfixturevalue(model_fixture)
     generator = generate.fsm(model, interegular.parse_pattern(pattern).to_fsm())
-    res = generator("test")
+    res = generator(**get_inputs(model_fixture))
     assert re.fullmatch(pattern, res) is not None, res
 
 
-@pytest.mark.parametrize("pattern", REGEX_PATTERNS)
+@pytest.mark.skip(
```

Comment:
There are a handful of open json validation issues. This is a good integration test case generally to address json generation failures because it applies random models to structured json generation.